portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,115 @@
-#content {
```

Comment:
A couples general comments for the CSS: can you alphabetize declarations (https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order) and make the spacing between rules consistent (https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation)?

If you want to keep the rules separated by page, consider adding section comments (https://google.github.io/styleguide/htmlcssguide.html#Section_Comments)